spec/javascripts/support/acuant.js

```diff
@@ -0,0 +1,26 @@
+import sinon from 'sinon';
+import { cleanup } from '@testing-library/react';
+
+export function useAcuant() {
+  afterEach(() => {
+    // While RTL will perform this automatically, it must to occur prior to
```

Comment:
I know this comment didn't change here, but it took me a hot second to realize this was React Testing Library and not right-to-left, WDYT about spelling this out?

Reply:
> I know this comment didn't change here, but it took me a hot second to realize this was React Testing Library and not right-to-left, WDYT about spelling this out?

Agreed 👍  As someone typically opposed to the use of acronyms, I'm a bit ashamed I let myself fall into this trap 😄

Reply:
> > I know this comment didn't change here, but it took me a hot second to realize this was React Testing Library and not right-to-left, WDYT about spelling this out?
> 
> Agreed 👍 As someone typically opposed to the use of acronyms, I'm a bit ashamed I let myself fall into this trap 😄

Updated in 89a42ab9b.